index.html

```diff
@@ -61,13 +61,162 @@ <h2 id="section__ownerRole">Software Developer</h2>
 					<img src="images/Vector4.svg" alt="" />
 				</section>
 				<ul class="contactsIcon">
-					<li><img src="images/Icons/Vectorgithubicon.svg" alt=""></li>
-                    <li><img src="images/Icons/Grouplinkedinicon.svg" alt=""></li>
-                    <li><img src="images/Icons/Vectorfinger.svg" alt=""></li>
-                    <li><img src="images/Icons/Vectortwittericon.svg" alt=""></li>
-                    <li><img src="images/Icons/VectorMicon.svg" alt=""></li>
+					<li><img src="images/Icons/Vectorgithubicon.svg" alt="" /></li>
+					<li><img src="images/Icons/Grouplinkedinicon.svg" alt="" /></li>
+					<li><img src="images/Icons/Vectorfinger.svg" alt="" /></li>
+					<li><img src="images/Icons/Vectortwittericon.svg" alt="" /></li>
+					<li><img src="images/Icons/VectorMicon.svg" alt="" /></li>
 				</ul>
 			</main>
 		</div>
+		<div class="projects-container">
+			<div class="work">
+				<img id="arrow" src="images/Group55.svg" alt="" />&nbsp;&nbsp;<span
+					id="work"
+					>Work</span
+				>
+			</div>
+			<p id="myrecentwork">MY RECENT WORK</p>
+			<img class="line" src="images/Vector4.svg" alt="" />
+			<div class="projects">
+				<div class="projects__item">
+					<img class="imgPlaceholder" src="images/ImgPlaceholder.svg" alt="" />
+					<h2 class="projects__item-name">Multi-post stories</h2>
+					<p class="projects__item-details">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<ul class="projects__item-language">
+						<li class="projects__item-language-item">css</li>
+						<li>
+							<img src="images/Rectangle.svg" alt="" />
+						</li>
+
+						<li class="projects__item-language-item">html</li>
+						<li>
+							<img src="images/Rectangle.svg" alt="" />
+						</li>
+						<li class="projects__item-language-item">bootstrap</li>
+						<li>
+							<img src="images/Rectangle.svg" alt="" />
+						</li>
+						<li class="projects__item-language-item">Ruby</li>
+					</ul>
+					<p class="seeProject">See project</p>
```

Comment:
- [ ] You are doing an excellent job, your portfolio is looking great! Let's make it even better by making this "See project" clickable, you could use an `a` link or a `button` tag. It would be great if you did this for every button.